app-backend/batch/src/main/scala/com/azavea/rf/batch/airflow/export/CreateExportDef.scala

```diff
@@ -0,0 +1,165 @@
+package com.azavea.rf.batch.airflow.export
+
+import java.util.UUID
+
+import scala.collection.JavaConverters._
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.{Await, Future}
+import scala.concurrent.duration._
+import scala.util._
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import cats.data._
+import cats.implicits._
+import com.amazonaws.auth.DefaultAWSCredentialsProviderChain
+import com.amazonaws.services.elasticmapreduce.{AmazonElasticMapReduceClient}
+import com.amazonaws.services.elasticmapreduce.model.{AddJobFlowStepsRequest, StepConfig, HadoopJarStepConfig}
+import com.amazonaws.services.route53._
+import com.amazonaws.services.route53.model._
+import io.circe.generic.JsonCodec
+import io.circe.syntax._
+
+import com.azavea.rf.batch.JobNoEC
+import com.azavea.rf.batch.util._
+import com.azavea.rf.database.{Database => DB}
+import com.azavea.rf.database.tables._
+import com.azavea.rf.datamodel._
+
+case class CreateExportDef(exportId: UUID)(implicit val database: DB) extends JobNoEC {
+  val name = "create_export_def"
+  implicit val system       = ActorSystem(s"$name-system")
+  implicit val materializer = ActorMaterializer()
+    
+  protected def writeExportDefToS3(exportDef: ExportDefinition) = {
```

Comment:
Makes sense to make a Future, as it's not the fastest operation, and it's highly parallelizable (this requires that execution context from `Job` trait).